rs/consensus/vetkd/src/lib.rs

```diff
@@ -204,78 +210,91 @@ impl VetKdPayloadBuilderImpl {
             )
         };
 
-        // Iterate over all outstanding VetKD requests
-        let mut candidates = BTreeMap::new();
-        let mut accumulated_size = 0;
-        for (callback_id, context) in state.signature_request_contexts() {
-            if !context.is_vetkd() {
-                // Skip non-vetkd contexts.
-                continue;
-            }
+        let accumulated_size_estimate = AtomicUsize::new(0);
```

Comment:
```suggestion
        let accumulated_size = AtomicUsize::new(0);
```
Since it tracks the actual accumulated size after the switch to `take_any_while` AFAIU